com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -500,6 +502,7 @@ private static Options getCommandLineOptions() {
 
         // debug options
         addOption(options, null, "debug-ne-upload", false, "Outputs the files sent to build server as upload.zip", false);
+        addOption(options, null, "debug-output-spirv", false, "Force build SPIR-V shaders", false);
```

Comment:
This is only needed for development when you have built an engine with the vulkan adapter (building with "--with-vulkan"). The alternative is to always use the editor, or appmanifest + extender, or setting the "output_spirv" in the game.project, but all of them require a bunch of extra steps that I'm not super fond of. We can also remove this option later I think, when we can set vulkan as default.